var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -398,7 +396,6 @@ class Trilinos(CMakePackage, CudaPackage):
 
     # Dependencies/conflicts when MPI is disabled
     depends_on('hdf5~mpi', when='+hdf5~mpi')
-    conflicts('+parmetis', when='~mpi')
```

Comment:
Maybe this was supposed to be `+metis`? But then again, removing this doesn't change the current behaviour, so I'm fine with it.

Reply:
I think the updated behavior is correct.